diff --git a/lib/albertcore/src/core/usagedatabase.cpp b/lib/albertcore/src/core/usagedatabase.cpp
index a5cd77b..a970ce4 100644
--- a/lib/albertcore/src/core/usagedatabase.cpp
+++ b/lib/albertcore/src/core/usagedatabase.cpp
@@ -78,7 +78,7 @@ void UsageDatabase::initialize() {
                 "    handler_id INTEGER NOT NULL REFERENCES query_handler(id) ON UPDATE CASCADE, "
                 "    runtime INTEGER NOT NULL, "
                 "    PRIMARY KEY (query_id, handler_id) "
-                ") WITHOUT ROWID; "))
+                "); "))
         qFatal("Unable to create table 'execution': %s", q.lastError().text().toUtf8().constData());
 
     if (!q.exec("CREATE TABLE IF NOT EXISTS activation ( "
